src/lib.rs

```diff
@@ -89,7 +89,7 @@ pub type GRefCell<T> = Gc<RefCell<T>>;
 pub type GMutex<T> = Gc<Mutex<T>>;
 
 /// A convenient alias for `Gc<RwLock<T>>`.
-/// Note that `Gc<Mutex<T>>` has additional specialized methods for working with `Mutex`s inside
+/// Note that `Gc<GRwLock<T>>` has additional specialized methods for working with `Mutex`s inside
```

Comment:
Ah thanks for the typo fix. I think this maybe should read `Gc<RwLock<T>>` though
See relevant impls: https://github.com/Others/shredder/blob/master/src/smart_ptr/gc.rs#L464